RadarSDK/Include/RadarTripOptions.h

```diff
@@ -57,7 +57,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, assign) RadarRouteMode mode;
 
-@property (nonatomic, assign) UInt8 approachingThreshold;
+@property (nonatomic, assign) UInt16 approachingThreshold;
```

Comment:
We don't limit this number server side, so I imagine we'd want to a) allow more bits, b) think about a server side limit